notebooks/COS/DataDl/DataDl.ipynb

```diff
@@ -160,7 +160,7 @@
     "\n",
```

Comment:
<p>Is it possible to opt for more descriptive variable&nbsp;names instead&nbsp;of query_1, query_2, and limq2?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>